package/gluon-radv-filterd/src/mac.h

```diff
@@ -0,0 +1,8 @@
+#include <stdint.h>
+#include <linux/if_ether.h>
+
+#define F_MAC "%02hhx:%02hhx:%02hhx:%02hhx:%02hhx:%02hhx"
+#define F_MAC_IGN "%*2x:%*2x:%*2x:%*2x:%*2x:%*2x"
+#define F_MAC_VAR(var) var[0], var[1], var[2], var[3], var[4], var[5]
```

Comment:
Enclosing each *var* in parentheses in the right-hand side of this defintion would be a good idea. Passing *&var* to this macro for scanf is very non-obvious macro magic, better use distinct definitions for printf and scanf arguments...